core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -6300,6 +6301,17 @@ private MyProjectRule(Class<? extends Project> clazz,
 
     sql(sql).with(HepProgram.builder().build()).withDecorrelation(true).checkUnchanged();
   }
+
+  /**
+   * Test case for <a href="https://issues.apache.org/jira/browse/CALCITE-3319">[CALCITE-3319]</a>
+   */
```

Comment:
We better include the JIRA title in the link.

Reply:
OK. I update the PR. Thanks for reminding me.